Issue44 chase

chase-scott opened the pull request:
Fixed some errors in the inital commit and began wallFollowing state.

Jakerem1 commented:
Why do so many files look like they were just deleted and replaced?

Jakerem1 commented on mail_delivery_robot/control/robotDriver.py:
```diff
@@ -175,6 +184,12 @@ def updateDistance(self, data):
 
```
There is no call to update the state here, so whats currently happening is only timer based instead of ROS interrupt based. Should this be how it works as it looks like it might not work too well.

Jakerem1 replied:
^ Line 175/184

chase-scott replied:
You're right, that will be added soon in the next pull.

chase-scott commented:
> Why do so many files look like they were just deleted and replaced?



> Why do so many files look like they were just deleted and replaced?

Seems to be an issue with VSCode git, I will work on resolving that in the future.
